hide_invoice_shipping_address/views/res_partner_view.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2024 Roger Sans <roger.sans@sygel.es>
+    License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <data>
+        <record model="ir.ui.view" id="res_partner_form_inherit">
+            <field name="name">res.partner.form.inherit</field>
+            <field name="model">res.partner</field>
+            <field name="inherit_id" ref="base.view_partner_form"/>
+            <field name="arch" type="xml">
+                <xpath expr="//group[@name='sale']" position="inside">
+                    <field name="invoice_shipping_address_print"/>
+                </xpath>
+
+                <xpath expr="//field[@name='child_ids']/form/sheet/group/group[2]" position="inside">
```

Comment:
I would not include this field here, as only the most basic information is shown in this view. WDYT?

Reply:
Done